src/lib/dnssd/TxtFields.cpp

```diff
@@ -195,62 +195,53 @@ TxtFieldKey GetTxtFieldKey(const ByteSpan & key)
+void FillNodeDataFromTxt(const ByteSpan & key, const ByteSpan & val, CommissionNodeData & nodeData)
 {
     TxtFieldKey keyType = Internal::GetTxtFieldKey(key);
     switch (keyType)
     {
     case TxtFieldKey::kLongDiscriminator:
-        nodeData.commissionData.longDiscriminator = Internal::GetLongDiscriminator(val);
+        nodeData.longDiscriminator = Internal::GetLongDiscriminator(val);
         break;
     case TxtFieldKey::kVendorProduct:
-        nodeData.commissionData.vendorId  = Internal::GetVendor(val);
-        nodeData.commissionData.productId = Internal::GetProduct(val);
+        nodeData.vendorId  = Internal::GetVendor(val);
+        nodeData.productId = Internal::GetProduct(val);
         break;
     case TxtFieldKey::kCommissioningMode:
-        nodeData.commissionData.commissioningMode = Internal::GetCommissioningMode(val);
+        nodeData.commissioningMode = Internal::GetCommissioningMode(val);
         break;
     case TxtFieldKey::kDeviceType:
-        nodeData.commissionData.deviceType = Internal::GetDeviceType(val);
+        nodeData.deviceType = Internal::GetDeviceType(val);
         break;
     case TxtFieldKey::kDeviceName:
-        Internal::GetDeviceName(val, nodeData.commissionData.deviceName);
+        Internal::GetDeviceName(val, nodeData.deviceName);
         break;
     case TxtFieldKey::kRotatingDeviceId:
-        Internal::GetRotatingDeviceId(val, nodeData.commissionData.rotatingId, &nodeData.commissionData.rotatingIdLen);
+        Internal::GetRotatingDeviceId(val, nodeData.rotatingId, &nodeData.rotatingIdLen);
         break;
     case TxtFieldKey::kPairingInstruction:
-        Internal::GetPairingInstruction(val, nodeData.commissionData.pairingInstruction);
+        Internal::GetPairingInstruction(val, nodeData.pairingInstruction);
         break;
     case TxtFieldKey::kPairingHint:
-        nodeData.commissionData.pairingHint = Internal::GetPairingHint(val);
-        break;
-    case TxtFieldKey::kSleepyIdleInterval:
-        nodeData.resolutionData.mrpRetryIntervalIdle = Internal::GetRetryInterval(val);
-        break;
-    case TxtFieldKey::kSleepyActiveInterval:
-        nodeData.resolutionData.mrpRetryIntervalActive = Internal::GetRetryInterval(val);
-        break;
-    case TxtFieldKey::kTcpSupported:
-        nodeData.resolutionData.supportsTcp = Internal::MakeBoolFromAsciiDecimal(val);
+        nodeData.pairingHint = Internal::GetPairingHint(val);
         break;
     default:
         break;
     }
 }
 
-void FillNodeDataFromTxt(const ByteSpan & key, const ByteSpan & value, ResolvedNodeData & nodeData)
+void FillNodeDataFromTxt(const ByteSpan & key, const ByteSpan & value, CommonResolutionData & nodeData)
 {
     switch (Internal::GetTxtFieldKey(key))
     {
     case TxtFieldKey::kSleepyIdleInterval:
-        nodeData.resolutionData.mrpRetryIntervalIdle = Internal::GetRetryInterval(value);
+        nodeData.mrpRetryIntervalIdle = Internal::GetRetryInterval(value);
```

Comment:
Looks like a rename was forgotten with the sleepy stuff. `mrpRetryInterfaceIdle` should be called `sleepyIdleInterval`.

Same for `mrpRetryIntervalActive` --> `sleepyActiveInterval`.

Can be done in a follow-up

Reply:
Agree, but not part of this PR. I believe that #17927 rename was at text field level and not at variable level.